webapp/components/features/FollowList.spec.js

```diff
@@ -0,0 +1,228 @@
+import { config, mount } from '@vue/test-utils'
+import Vuex from 'vuex'
+import FollowList from './FollowList.vue'
+
+const localVue = global.localVue
+
+config.stubs['client-only'] = '<span><slot /></span>'
+config.stubs['ds-space'] = '<span><slot /></span>'
+config.stubs['nuxt-link'] = '<span><slot /></span>'
+
+describe('FollowList.vue', () => {
+  let store, mocks, getters, propsData
+
+  beforeAll(() => {
+    mocks = {
+      $t: jest.fn(),
+    }
+    getters = {
+      'auth/user': () => {
+        return {}
+      },
+      'auth/isModerator': () => false,
+    }
+    propsData = { user }
+  })
+
+  describe('mount', () => {
+    beforeAll(() => {
+      store = new Vuex.Store({
+        getters,
+      })
+    })
+
+    describe('given a user with connections', () => {
+      ;['following', 'followedBy'].forEach(type =>
+        describe(`and type=${type}`, () => {
+          let wrapper
+          let queryMock
+
+          beforeAll(() => {
+            queryMock = jest.fn().mockResolvedValue({
+              data: { User: [{ [type]: additionalConnections[type] }] },
+            })
+
+            wrapper = mount(FollowList, {
+              store,
+              propsData: { ...propsData, type: type },
+              mocks: {
+                ...mocks,
+                $apollo: {
+                  query: queryMock,
+                },
+              },
+              localVue,
+            })
+          })
+
+          it(`shows the users ${type}`, () => {
+            expect(wrapper.findAll('.user-teaser').length).toEqual(user[type].length)
+          })
+          it(`has a button to load all remaining users ${type}`, async () => {
+            wrapper.find('button').trigger('click')
+            //await wrapper.vm.$nextTick()
```

Comment:
```suggestion
            await Vue.nextTick()
```
you must `import Vue from 'vue'` then run this :point_up_2:

Reply:
It's available both ways.
It's literally just switching from node <12 to node >=12 that makes the difference. I can reproduce that locally.

Reply:
ok, I haven't seen it used the way you added it, we have it `34` different places as `Vue.nextTick()` and as you noted, we use node version `lts`, which is `12`, so it doesn't make sense that it would work in all the other places and not for you.

Unfortunately, I am dealing with issues locally, but I will try to fix them and pull it down locally.

Reply:
Same with `Vue.nextTick`. What really bugs me is that I couldn't even reproduce that with a clean Vue project.

![passing-at-node-11](https://user-images.githubusercontent.com/3411649/77311303-1cde9c00-6d00-11ea-9e0b-9f32dfaaada9.png)
![failing-at-node-12](https://user-images.githubusercontent.com/3411649/77311304-1d773280-6d00-11ea-855b-1dca7269f504.png)

Reply:
fixed in 0b91601